src/components/main/StoreDetailSection/index.tsx

```diff
@@ -30,7 +30,9 @@ export default function StoreDetailSection({
     storeId: storeId ?? (Number(searchParams.get('storeId')) || undefined),
   });
 
-  const { data: reportData } = useGetReport(storeId ?? null);
+  const { data: reportData } = useGetReport(
+    storeId ?? Number(searchParams.get('storeId') || undefined),
```

Comment:
이러면 storeId가 없을때 NaN 나올거같은데

```suggestion
storeId ?? (Number(searchParams.get('storeId')) || null),
```
이렇게 쓰는건 어떨까여??